frontend/src/lib/components/ActivityLog/activityLogLogic.tsx

```diff
@@ -104,7 +104,7 @@ export const activityLogLogic = kea<activityLogLogicType>([
     })),
     listeners(({ actions }) => ({
         setPage: async (_, breakpoint) => {
-            await breakpoint()
+            breakpoint()
```

Comment:
Is this not the opposite of what we want? 🤔

Reply:
This is actually a bit tricky and eslint gets it right, which is that breakpoints without a delay are non-promises (there's nothing asynchronous going on then)